docs/maintaining-a-track.md

```diff
@@ -0,0 +1,88 @@
+# Maintaining an Exercism Language Track
+
+* [Philosophy](#philosophy)
+
+  * [Be Kind](#be-kind)
+  * [Be Responsive](#be-responsive)
+  * [Empower New Contributors](#empower-new-contributors)
+  * [Recognize Contributions](#recognize-contributions)
+  * [Praise in Public, Reprimand in Private](#praise-in-public-reprimand-in-private)
+  * [Improve the Learning Experience](#improve-the-learning-experience)
+  * [Collaborate](#collaborate)
+
+* [Day-to-Day Tactics](#day-to-day-tactics)
+
+  * [Watch Key Repositories](#watch-key-repositories)
+  * [Perform Common Tasks](#perform-common-tasks)
+
+## Philosophy
+
+### Be Kind
+
+First and foremost, strive always to be kind, humble and show gratitude for the work being contributed.
+
+**_Kind_ is not the same thing as _nice_.** Being kind does not mean being a pushover.
+Doing the right thing is not always the thing that pleases the most people.
+
+### Be Responsive
+
+Even if you do not have an answer immediately, respond quickly. This lets the person on the other end know that they've been heard. If it takes a long time to figure out what to do, be honest about what's going on.
```

Comment:
This is such a good point. I've been doing this more and more since joining GitHub (still trying to get ahold of it).